crates/core/tests/dom.rs

```diff
@@ -61,6 +61,7 @@ fn dom_builder_example() {
     assert_eq!(buffer.as_str(), expected);
 }
 
+/*
```

Comment:
Remove commented out test or add TODO to fix.

Reply:
https://github.com/liveview-native/liveview-native-core/issues/58 I authored a cleanup issue and referenced it.